client/src/app/shared/components/list-of-speakers-content/list-of-speakers-content.component.html

```diff
@@ -316,12 +317,12 @@ <h1 class="los-title" *ngIf="!customTitle">
             </button>
 
             <!-- star button -->
-            <button mat-menu-item (click)="onMarkButton(speaker)">
+            <button mat-menu-item (click)="onMarkButton(speaker)" *ngIf="canSetMarkSelf">
```

Comment:
So an admin cannot set the marked button, if the set-self config is off, even for other users?

Reply:
done - reverted